pyspark-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,11 +1,11 @@
 """Project settings. There is no need to edit this file unless you want to change values
 from the Kedro defaults. For further information, including these default values, see
 https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
-from {{cookiecutter.python_package}}.context import ProjectContext
 
 # Instantiated project hooks.
-# from {{ cookiecutter.python_package }}.hooks import ProjectHooks
-# HOOKS = (ProjectHooks(),)
+from {{cookiecutter.python_package}}.hooks import SparkHooks
+
+HOOKS = (SparkHooks(),)
```

Comment:
```suggestion
from {{cookiecutter.python_package}}.hooks import SparkHooks
HOOKS = (SparkHooks(),)
```
(extremely pedantic I know 😆)

Reply:
Linters seem to not like this change 😅

Reply:
Ah yes, ok. Don't do it then 😀